controllers/component_controller.go

```diff
@@ -548,12 +545,6 @@ func (r *ComponentReconciler) generateGitops(ctx context.Context, req ctrl.Reque
 		return err
 	}
 
-	// GenerateTektonBuild returns a sanitized error message
-	err = appservicegitops.GenerateTektonBuild(tempDir, *component, r.AppFS, gitOpsContext, gitopsConfig)
-	if err != nil {
-		log.Error(err, "unable to generate gitops build resources due to error")
-		return err
-	}
 	//Gitops functions return sanitized error messages
 	err = r.Generator.CommitAndPush(tempDir, "", gitOpsURL, mappedGitOpsComponent.Name, gitOpsBranch, "Generating Tekton resources")
```

Comment:
Ah, missed this. Can you actually change this line here to say "Generated GitOps Resources"?

Reply:
updated